service/src/test/java/bio/terra/pipelines/configuration/external/IngressConfigurationTest.java

```diff
@@ -0,0 +1,17 @@
+package bio.terra.pipelines.configuration.external;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+import bio.terra.pipelines.app.configuration.external.IngressConfiguration;
+import bio.terra.pipelines.testutils.BaseEmbeddedDbTest;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+
+class IngressConfigurationTest extends BaseEmbeddedDbTest {
+  @Autowired IngressConfiguration ingressConfiguration;
+
+  @Test
+  void verifyIngressConfiguration() {
+    assertNotNull(ingressConfiguration.getDomainName());
```

Comment:
you could also put a value in the application-test.yml and test against that value here if you want

Reply:
nice, good call!